components/builder-originsrv/src/migrations/2018-02-21-142803_add_origin_encryption_keys/up.sql

```diff
@@ -0,0 +1,111 @@
+CREATE SEQUENCE IF NOT EXISTS origin_public_encryption_key_id_seq;
+
+CREATE TABLE IF NOT EXISTS origin_public_encryption_keys (
+  id bigint PRIMARY KEY DEFAULT next_id_v1('origin_public_key_id_seq'),
+  origin_id bigint REFERENCES origins(id),
+  owner_id bigint,
+  name text,
+  revision text,
+  full_name text UNIQUE,
+  body bytea,
+  created_at timestamptz DEFAULT now(),
+  updated_at timestamptz DEFAULT now()
+);
+
+CREATE OR REPLACE FUNCTION insert_origin_public_encryption_key_v1 (
+  opek_origin_id bigint,
+  opek_owner_id bigint,
+  opek_name text,
+  opek_revision text,
+  opek_full_name text,
+  opek_body bytea
+) RETURNS SETOF origin_public_encryption_keys AS $$
+    BEGIN
```

Comment:
There was a discussion a while ago that we should try to use SQL instead of PSQL as the language wherever possible.. I don't think it's a blocker but just FYI...